Microsoft.HealthVault.Fhir/Constants/HealthVaultExtensions.cs

```diff
@@ -59,5 +59,7 @@ public static class HealthVaultExtensions
         public const string PatientBirthTime = HealthVaultExtensionsUri + "birth-time";
         public const string PatientTitle = "codable-title";
         public const string PatientSuffix = "codable-suffix";
+
+        public const string Organisation = HealthVaultExtensionsUri + "Organisation";
```

Comment:
>Organisation [](start = 71, length = 12)

user all lowercase like the rest of the extension names. Also, consider using the American English spelling: organization.